script/Enums/ScrollDirection.lua

```diff
@@ -0,0 +1,6 @@
+---@enum
+Enums.ScrollDirection = {
```

Comment:
Enums.UI.ScrollDirection >> rename file to UIEnums.lua 

Think that would be better

Reply:
done